golang/cmd/data-bridge/main.go

```diff
@@ -138,6 +139,16 @@ func main() {
 				zap.S().Errorf("failed to create mqtt client: %s", err)
 			}
 		}
+	case 2: // mqtt to mqtt
+		zap.S().Infof("Starting mqtt to mqtt bridge")
+		clientA, err = newMqttClient(brokerA, topic, false, "")
+		if err != nil {
+			zap.S().Errorf("failed to create mqtt client: %s", err)
+		}
+		clientB, err = newMqttClient(brokerB, topic, false, "")
+		if err != nil {
+			zap.S().Errorf("failed to create mqtt client: %s", err)
+		}
```

Comment:
Case 3: mqtt to kafka ?

Reply:
can be done in case 2, setting the correct bridge-mode